CHANGELOG.md

```diff
@@ -38,4 +43,4 @@ Updated Rubocop Version
 
 ### Removed
 
-- Gemfile.lock from .gitignore
\ No newline at end of file
+- Gemfile.lock from .gitignore
```

Comment:
What??? These two lines look the same to my eyes 👀

Reply:
Same for me. My editor did it automatically. Probably just adds a newline by default to the end of the file because there wasn't any before.